src/api/manifest/ManifestUtil.ts

```diff
@@ -3,7 +3,7 @@ import { readFile, writeFile } from 'fs-extra'
 import path from 'path'
 import { memoize } from 'ramda'
 
-import { CommandError } from './errors'
+import { CommandError } from '../error/errors'
 
 const readFileUtf = (file: string): Promise<string> => {
```

Comment:
These manifest utils are old, they should be replaced by the ManifestEditor class (also, the ManifestEditor class should be renamed to just Manifest)

Reply:
So there's no need for these old manifest utils to be exported on the api

Reply:
OK, just removed from the export